.travis.yml

```diff
@@ -126,6 +116,6 @@ deploy:
   on:
     rust: nightly
     tags: true
-    condition: $GCC_VER = 7
+    condition: $CLANG_CXX = clang++-7 OR $CLANG_CXX = clang++
```

Comment:
We should only deploy in one config for Linux and in one for Mac, with this condition we're deploying multiple times

Reply:
it was already the case before this patch, no ?

Reply:
but yes I agree

Reply:
oh yes you're right